source/install/install-ubuntu-2004-postgresql.rst

```diff
@@ -15,39 +15,36 @@ Assume that the IP address of this server is 10.10.10.1.
 
   When the installation is complete, the PostgreSQL server is running, and a Linux user account called *postgres* has been created.
 
-2. Log in to the *postgres* account.
+2. Access postgres with one of the below options:
 
-  ``sudo --login --user postgres``
-
-3. Start the PostgreSQL interactive terminal.
-
-  ``sudo -u postgres psql``
-
-4.  Create the Mattermost database.
+  a.  ``sudo --login --user postgres`` then ``psql``
```

Comment:
Question: Do these 2 ways of accessing the PostgreSQL terminal apply to all of our supported operating systems, including Ubuntu 18.0.4 and 16.0.4, Debian Buster, and RHEL 7 & 8? If yes, we'll want to apply this update for each applicable section of the docs. Would you be open to making these additional updates as well?

Reply:
For point 1, I thought about bullets but on step 11 (the last one) we use letters instead of bullets for two options. 

Point 2: I looked through RHEL 8 / RHEL 7 / Debian / and the other Ubuntu's and it just walks you though the method of switching to the postgres user.

Reply:
Happy to change it to bullets though for clarity!

Reply:
Thank you! I appreciate you confirming that other updates aren't needed. Please feel free to update the other section of that same page where we're also using numbered steps rather than bullets for consistency.

Reply:
Done, you should see it in the latest commit!